tox.ini

```diff
@@ -107,6 +107,7 @@ envlist =
     sqlalchemy_contrib-{py27,py34,py35,py36,py37}-sqlalchemy{10,11,12}-psycopg228-mysqlconnector
     sqlite3_contrib-{py27,py34,py35,py36,py37}-sqlite3
     tornado_contrib-{py27,py34,py35,py36,py37}-tornado{40,41,42,43,44,45}
+    tornado_contrib-{py37}-tornado{50,51,60}
     tornado_contrib-{py27}-tornado{40,41,42,43,44,45}-futures{30,31,32}
```

Comment:
Do we need to test with `futures` for `5x`/`6x`?

Reply:
`futures` were only used for Tornado tests with `py27`. The standard lib `futures` is wrapped by default:

https://github.com/DataDog/dd-trace-py/blob/majorgreys/tornado60/ddtrace/contrib/tornado/patch.py#L32